src/context/AppProvider/AppContext.tsx

```diff
@@ -154,6 +154,10 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
           walletId: await wallet.getDeviceID(),
         }),
       )
+
+      for (const accountId of Object.keys(accountMetadataByAccountId)) {
+        dispatch(portfolio.actions.enableAccountId(accountId))
+      }
```

Comment:
high risk here

Reply:
Wrap this in a feature flag check @woodenfurniture